dashboard/app/models/ability.rb

```diff
@@ -130,7 +130,9 @@ def initialize(user)
       end
 
       if user.permission?(UserPermission::PLC_REVIEWER)
+        can :manage, PeerReview
         can :index_escalated, :peer_review_submissions
+        can :dashboard, :peer_reviews
```

Comment:
should this be `PeerReview`?

Reply:
No - the api controller is peer_reviews_controller, not peer_review_controller